glide.yaml

```diff
@@ -0,0 +1,32 @@
+package: github.com/almighty/almighty-core
+import:
+- package: github.com/dgrijalva/jwt-go
+  version: ^3.0.0
+- package: github.com/goadesign/goa
+  version: master
+  vcs: git
+  subpackages:
+  - client
+  - cors
+  - design
+  - design/apidsl
+  - goagen
+  - goatest
+  - middleware
+  - middleware/security/jwt
+- package: github.com/lsegal/gucumber
```

Comment:
- package: golang.org/x/tools
  subpackages:
  - go/ast/astutil
- package: gopkg.in/yaml.v2
- package: bitbucket.org/pkg/inflect
- package: github.com/kr/pretty
- package: github.com/kr/text